source/fundamentals/indexes.txt

```diff
@@ -244,10 +244,10 @@ each of the preceding methods.
 Create a Search Index
 +++++++++++++++++++++
 
-You can use the `createSearchIndex() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#createSearchIndex(org.bson.conversions.Bson)>`__
-and the
-`createSearchIndexes() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#createSearchIndexes(java.util.List)>`__
-methods to create Atlas Search and Vector Search indexes.
+You can use the `createSearchIndex()
+<{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoCollection.html#createSearchIndex(org.bson.conversions.Bson)>`__
+method to create an Atlas Search index. You *cannot* use this method to create a
+Vector Search index at this time.
```

Comment:
S: I don't think we need "at this time" since it might make people think it's coming soon. Instead we can just change this page it if it does change in the future
```suggestion
Vector Search index.
```